src/csexp_rpc/csexp_rpc.ml

```diff
@@ -15,6 +15,73 @@ end
 
 module Session_id = Id.Make ()
 
+module Socket = struct
+  module type Unix_socket = sig
+    val connect : Unix.file_descr -> socket:string -> unit
+
+    val bind : Unix.file_descr -> socket:string -> unit
+  end
+
+  module U = Unix
```

Comment:
We should expand this to avoid breaking if someone adds an optional argument to `Unix.connect` or `Unix.bind`:

```suggestion
  module U =  struct
    let connect fd ~socket = Unix.connect fd ~socket
    let bind fd ~socket = Unix.bind fd ~socket
  end
```

Reply:
done.